app/page/template/content/collection-details.htm

```diff
@@ -0,0 +1,64 @@
+<div id="collection-details"
+     class="collection content-wrapper"
+     data-bind="with: $root.collection_details, removed_from_view: $root.collection_details.removed_from_view">
+
+  <!-- ko if: conversation_et() -->
+
+    <div class="content-titlebar">
+
+      <div class="content-titlebar-items-left">
+        <span class="content-titlebar-icon icon-back" data-bind="click: click_on_back_button"></span>
+      </div>
+
+      <span class="content-titlebar-items-center">
+        <span><span class="label-bold-xs">Pictures</span> <span class="text-graphite">in</span> <span class="text-graphite" data-bind="text: conversation_et().display_name()"></span></span>
```

Comment:
Ensure strings are localizable and make use of the current template type for the correct naming.